common-util/src/test/java/org/sunbird/common/models/AppTest.java

```diff
@@ -2,192 +2,100 @@
-    assmntMap2.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap2.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, Object> assmntMap3 = new HashMap<>();
-    assmntMap3.put(JsonKey.ASSESSMENT_SCORE, "8");
-    assmntMap3.put(JsonKey.ASSESSMENT_MAX_SCORE, "10");
-    assmntMap3.put(JsonKey.ASSESSMENT_ITEM_ID, "3");
-    assmntMap3.put(JsonKey.COURSE_ID, "CSR1");
-    assmntMap3.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap3.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, Object> assmntMap4 = new HashMap<>();
-    assmntMap4.put(JsonKey.ASSESSMENT_SCORE, "5");
-    assmntMap4.put(JsonKey.ASSESSMENT_MAX_SCORE, "10");
-    assmntMap4.put(JsonKey.ASSESSMENT_ITEM_ID, "4");
-    assmntMap4.put(JsonKey.COURSE_ID, "CSR1");
-    assmntMap4.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap4.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, Object> assmntMap5 = new HashMap<>();
-    assmntMap5.put(JsonKey.ASSESSMENT_SCORE, "5");
-    assmntMap5.put(JsonKey.ASSESSMENT_MAX_SCORE, "10");
-    assmntMap5.put(JsonKey.ASSESSMENT_ITEM_ID, "5");
-    assmntMap5.put(JsonKey.COURSE_ID, "CSR1");
-    assmntMap5.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap5.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, Object> assmntMap6 = new HashMap<>();
-    assmntMap6.put(JsonKey.ASSESSMENT_SCORE, "6");
-    assmntMap6.put(JsonKey.ASSESSMENT_MAX_SCORE, "10");
-    assmntMap6.put(JsonKey.ASSESSMENT_ITEM_ID, "6");
-    assmntMap6.put(JsonKey.COURSE_ID, "CSR1");
-    assmntMap6.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap6.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, Object> assmntMap7 = new HashMap<>();
-    assmntMap7.put(JsonKey.ASSESSMENT_SCORE, "8");
-    assmntMap7.put(JsonKey.ASSESSMENT_MAX_SCORE, "10");
-    assmntMap7.put(JsonKey.ASSESSMENT_ITEM_ID, "7");
-    assmntMap7.put(JsonKey.COURSE_ID, "CSR1");
-    assmntMap7.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap7.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, Object> assmntMap8 = new HashMap<>();
-    assmntMap8.put(JsonKey.ASSESSMENT_SCORE, "9");
-    assmntMap8.put(JsonKey.ASSESSMENT_MAX_SCORE, "10");
-    assmntMap8.put(JsonKey.ASSESSMENT_ITEM_ID, "8");
-    assmntMap8.put(JsonKey.COURSE_ID, "CSR1");
-    assmntMap8.put(JsonKey.CONTENT_ID, "CON1");
-    assmntMap8.put(JsonKey.USER_ID, "USR1");
-
-    Map<String, List<Map<String, Object>>> map = new HashMap<>();
-
-    List<Map<String, Object>> list = new ArrayList<>();
-    list.add(assmntMap1);
-    list.add(assmntMap2);
-    list.add(assmntMap3);
-    list.add(assmntMap4);
-    list.add(assmntMap5);
-    list.add(assmntMap6);
-    list.add(assmntMap7);
-    list.add(assmntMap8);
-
-    map.put("USR1", list);
-
-    return map;
-  }
+public class AppTest extends BaseForHttpTest {
+	private static final String data = "{\"request\": { \"search\": {\"contentType\": [\"Story\"] }}}";
+	private static Map<String, String> headers = new HashMap<String, String>();
+
+	@BeforeClass
+	public static void init() {
+		headers.put("content-type", "application/json");
```

Comment:
Code changes are not properly formatted using scripts/setup.sh. Get it done with help from Manzarul.

Reply:
This applies to all Java files in this PR.